nilearn/image/tests/test_image.py

```diff
@@ -354,6 +360,15 @@ def test_index_img():
             'out of bounds|invalid index|out of range|boolean index',
             image.index_img, img_4d, i)
 
+    # if pandas is in the testing environment
+    if 'pandas' in sys.modules:
+        arr = np.array(np.random.rand(fourth_dim_size) > 0.5)
```

Comment:
We try really hard never to use the global random number generator in our test (as this creates a side effect). Inside a test, we do:
<pre>
rng = np.random.RandomState(0)
</pre>
and use rng instead of np.random (it has only a subset of the methods).

Also, the call to "np.array" is not necessary.